t/test_multivalue_separator.rb

```diff
@@ -68,4 +69,11 @@
     links.first[:url].must_equal 'http://parlinfo.aph.gov.au/parlInfo/search/display/display.w3p;query%3DId%3A%22handbook%2Fallmps%2FAK6%22'
     links.count.must_equal 1
   end
+
+  it 'should split URLs when there is space around the separator' do
+    url1 = { note: 'website', url: 'http://clarke.example.org' }
+    url2 = { note: 'website', url: 'https://www.conservatives.com/OurTeam/Duncan_Clarke' }
+    clarke[:links].must_include url1
+    clarke[:links].must_include url2
+  end
```

Comment:
I'd be a little more comfortable if this tested that the order gets maintained. Maybe something like:

```ruby
    urls = clarke[:links].select { |l| l[:note] == 'website' }.map { |l| l[:url] }
    urls.first.must_equal 'http://clarke.example.org'
    urls.last.must_equal 'https://www.conservatives.com/OurTeam/Duncan_Clarke'
```